pom.xml

```diff
@@ -89,36 +93,26 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <version>1.0.0</version>
+                <configuration>
+                    <to>
```

Comment:
This is the standardization I have noticed @dgvigil start to do across pom files:
https://github.com/racker/salus-app-base/blob/master/pom.xml#L106-L113